genbindings.sh

```diff
@@ -64,12 +65,14 @@ fi
 
 rm genbindings_path_map_test_file.c
 
-ENV_TARGET=$(rustc --version --verbose | grep host | awk '{ print $2 }' | sed 's/-/_/g')
 case "$ENV_TARGET" in
 	"x86_64"*)
 		export RUSTFLAGS="$BASE_RUSTFLAGS -C target-cpu=sandybridge"
 		export CFLAGS_$ENV_TARGET="$BASE_HOST_CFLAGS -march=sandybridge -mcpu=sandybridge -mtune=sandybridge"
 		;;
+	"aarch64-apple"*)
+		export RUSTFLAGS="$BASE_RUSTFLAGS -C target-cpu=apple-a14"
+		export CFLAGS_$ENV_TARGET="$BASE_HOST_CFLAGS -mcpu=apple-a14"
```

Comment:
I think you're missing a `;;` after this line:
```suggestion
		export CFLAGS_$ENV_TARGET="$BASE_HOST_CFLAGS -mcpu=apple-a14"
		;;
```